assets/src/application/ui/layout/espressoTable/TableHeader.tsx

```diff
@@ -46,6 +46,19 @@ const TableHeader: React.FC<TableHeaderProps> = ({ headerRows, tableId, ...props
 							</TableHeaderCell>
 						);
 					})}
+
+					{showDragHandle && (
+						<TableHeaderCell
+							className={props.className}
+							colNumber={headerRow.cells.length}
+							key={`row-${row}-col-${headerRow.cells.length}`}
+							rowNumber={row}
+							rowType={RowType.header}
+							id={`${tableId}-header-cell-draghandle`}
+						>
+							{''}
```

Comment:
why empty string is needed here, is `children` required?

Reply:
Drag handle does not need a heading IMHO

Reply:
removed